pages/linux/sar.md

```diff
@@ -0,0 +1,27 @@
+# sar
+
+> Monitor performance of various Linux subsystems.
+
+- Report I/O and Transfer rate every 1 second:
+
+`sar -b 1`
```

Comment:
There aren't any long forms of the options?

Reply:
unfortunately not :(